benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -141,6 +141,13 @@ export default class Overview extends React.Component {
     }
   }
 
+  addTypeToFilter = (filters) =>
+    filters.forEach((filter) => {
+      const [runsetIdx, name, columnIdx] = filter.id.split("_");
```

Comment:
For reference: #731

Reply:
Yes I agree, there are a lot of things that could be refactored. I assume we leave it for now in this PR and handle this separately?

Reply:
Probably better that way, that's why I created the issue.